IntelFsp2Pkg/Library/BaseFspCommonLib/FspCommonLib.c

```diff
@@ -90,8 +90,18 @@ GetFspGlobalDataPointer (
   )
 {
   FSP_GLOBAL_DATA  *FspData;
-
+  UINTN            FspDataAddress;

```

Comment:
I prefer that we change SetFspGlobalDataPointer() to below:
  *((volatile UINTN *)(UINTN)PcdGet32 (PcdGlobalDataPointerAddress)) = (UINTN)FspData;

Reply:
Yes, this change will be better.
But from compatibility side, there are many code in OneSiliconPkg and IntelFsp2Pkg with (UINTN)FspData == 0xFFFFFFFF or (UINTN)FspData != 0xFFFFFFFF for comparing.